Extension/bin/messages/de/messages.json

```diff
@@ -1695,8 +1695,8 @@
   null,
   "Für die Vektoroperation sind identische Elementtypen erforderlich.",
   "Die Vektoroperation gilt nicht für Vektoren mit nicht integralem Typ.",
-  "Die Datei \"%s\" kann nicht geöffnet werden: %sq2.",
-  "Die -Datei \"%s\" kann nicht geöffnet werden: %sq2: %s3.",
+  "Die Datei %s kann nicht geöffnet werden: %sq2.",
```

Comment:
I believe this should be `Quelldatei` -- I think we may need code changes to properly translate this. I've filed an internal issue.

Reply:
...but this may not block the check in if we can't get it figured out soon enough, i.e. it could be fixed in a follow up PR

Reply:
We can take a look at this in another PR